docs/tutorials/01_KalmanFilterTutorial.py

```diff
@@ -2,298 +2,195 @@
 transition_model.covar(time_interval=timedelta(seconds=1))
 
 # %%
-# Create Kalman predictor
-# ---------------------------------
-from stonesoup.predictor.kalman import KalmanPredictor
-predictor = KalmanPredictor(transition_model)
+# At this point you can play with the various parameters and see how it effects the simulated
+# output.
 
 # %%
-# Simulate measurement
-# --------------------
-# Given a prediction of the target's state and covariance at time :math:`k`, we receive a
-# measurement from a sensor (by weighing our "trust" in our own prediciton, and that of the
-# sensor's measurement, we can provide a refined estimate of the target's state at time :math:`k`).
-#
-# Obviously, a measurement depends on the actual state of the system. We can infer that, given our
-# prediciton, :math:`\exists` some mapping :math:`H_{k}:\ state\mapsto\ measurement` such that we
-# can define a distribution of possible measurements we might get (and corresponding expected
-# covariance), given :math:`\hat{x}_{k}`:
-#
-# .. math::
-#       \vec{\mu}_{expected} &= H_{k}\hat{x}_{k}\\
-#          \Sigma_{expected} &= H_{k}P_{k}H_{k}^T
+# (Cup of tea icon)
 
 # %%
-# Simulate measurement errors
-# ---------------------------
+# Simulate measurements
+# ^^^^^^^^^^^^^^^^^^^^^
 #
-# Adjusting our measurement in the same manner to the predicted state gives:
+# We'll use one of Stone Soup's measurement models in order to generate
+# measurements from the ground truth. For the moment we assume a sensor which can detect the
+# position, but not velocity, of a target. We're going to need a :class:`~.Detection` type to
+# store the detections, and a :class:`~.LinearGaussian` measurement model.
+from stonesoup.types.detection import Detection
+from stonesoup.models.measurement.linear import LinearGaussian
+
+# %%
+# The measurement model is set up by indicating the number of dimensions in the state vector and
+# the dimensions that are measured. Also specified is the covariance of the noise (which is additive
+# Gaussian), and for the present we assume homogenous and independent noise, so
 #
 # .. math::
-#       \vec{\mu}_{expected} &= H_{k}\hat{x}_{k}\\
-#          \Sigma_{expected} &= H_{k}P_{k}H_{k}^T + R_{k}
 #
-# accounting for some noise :math:`R_{k}`.
-# The Gaussian distribution defined by this has a mean equal to the detection/measurement received
-# from the sensor.
+#           R = \begin{bmatrix}
+#                   1 & 0\\
+#                     0 & 1\\
+#                \end{bmatrix} \omega
 #
-# Next we'll create our model which will describe errors of measuring in the observable state, and
-# how the hidden state maps to observable state. In this case we can simply map the :math:`x` and
-# :math:`y` directly.
-from stonesoup.types.detection import Detection
-from stonesoup.models.measurement.linear import LinearGaussian
+# where :math:`\omega` is set to 0.75 initially (but again, feel free to play around).
 measurement_model = LinearGaussian(
```

Comment:
(something I probably should have done) could add kwargs here to make it more clear.

Reply:
Not sure what you mean, but go ahead